src/pages/Fox/components/FoxWifHatHoldingLine.tsx

```diff
@@ -0,0 +1,87 @@
+import type { StackDirection } from '@chakra-ui/react'
+import { Button, Flex, HStack, Stack, Text, useColorModeValue } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletIcon } from 'components/Icons/WalletIcon'
+import { middleEllipsis } from 'lib/utils'
+
+type FoxWifHatHoldingLineProps = {
+  accountId: string
+  accountNumber: number
```

Comment:
suggestion: `accountNumber` can be derived with `selectAccountNumberByAccountId`, prefer to keep props minimal and derive here vs. passing it down

Reply:
Nice, done in [a79f134](https://github.com/shapeshift/web/pull/8605/commits/a79f1341343395bca74b469a901a3a4e12a71be0)